### ISSUE TITLE
Value gap between both DMP_TFs in Table1 and Table2

### ISSUE THREAD

**Issue by JongKook-Heo** [dual-view-molecule-pretraining/dmp#2]
### Before asking:   
1. search the issues.   
2. search the docs.    

#### What is your question?
Hi, @dual-view-molecule-pretraining , thanks for sharing your nice works!
I have some questions about result tables on your paper.
I am wondering what kind of difference exists between official test set from DeepChem and those of GROVER?
I found that there's a huge gap between both DMP_TFs, which are in Table1 and Table2 respectively. (i.e. BBBP, SIDER, ClinTox)

Best regards,

#### What's your environment?

 - fairseq Version (e.g., 1.0 or master):
 - PyTorch Version (e.g., 1.0)
 - OS (e.g., Linux):
 - How you installed fairseq (`pip`, source):
 - Build command you used (if compiling from source):
 - Python version:
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:
